README.md

```diff
@@ -117,7 +117,7 @@ Parse Server is continuously tested with the most recent releases of Node.js to
 | Node.js 12 | 12.22.7        | April 2022  | ✅ Yes        |
 | Node.js 14 | 14.18.1        | April 2023  | ✅ Yes        |
 | Node.js 15 | 15.14.0        | June 2021   | ✅ Yes        |
```

Comment:
Could you remove the line about Node 15 support?

Reply:
Done